From: Santiago Vila <sanvila@debian.org>
Subject: Do not use or define COMPILEDATE, it makes the build not to be reproducible

--- a/bin/yadifa/yadifa-config.c
+++ b/bin/yadifa/yadifa-config.c
@@ -385,10 +385,10 @@
             osformatln(termout, "%s %s (%s)", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE);
             break;
         case 2:
-            osformatln(termout, "%s %s (released %s, compiled %s)", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE, COMPILEDATE);
+            osformatln(termout, "%s %s (released %s)", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE);
             break;
         case 3:
-            osformatln(termout, "%s %s (released %s, compiled %s)", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE, COMPILEDATE);
+            osformatln(termout, "%s %s (released %s)", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE);
             yadifa_print_authors();
             break;
         default:
--- a/lib/dnscore/src/dnscore.c
+++ b/lib/dnscore/src/dnscore.c
@@ -103,9 +103,9 @@
 #endif
 
 #ifdef DEBUG
-const char *dnscore_lib = "dnscore " __DATE__ " " __TIME__ " debug";
+const char *dnscore_lib = "dnscore debug";
 #else
-const char *dnscore_lib = "dnscore " __DATE__ " " __TIME__ " release";
+const char *dnscore_lib = "dnscore release";
 #endif
 
 static const char* ARCH_RECOMPILE_WARNING = "Please recompile with the correct settings.";
--- a/lib/dnsdb/src/zdb.c
+++ b/lib/dnsdb/src/zdb.c
@@ -92,9 +92,9 @@
 #endif
 
 #ifdef DEBUG
-const char *dnsdb_lib = "dnsdb " __DATE__ " " __TIME__ " debug";
+const char *dnsdb_lib = "dnsdb debug";
 #else
-const char *dnsdb_lib = "dnsdb " __DATE__ " " __TIME__ " release";
+const char *dnsdb_lib = "dnsdb release";
 #endif
 
 void dnssec_keystore_init();
--- a/lib/dnszone/src/dnszone.c
+++ b/lib/dnszone/src/dnszone.c
@@ -64,9 +64,9 @@
 #endif
 
 #ifdef DEBUG
-const char *dnszone_lib = "dnszone " __DATE__ " " __TIME__ " debug";
+const char *dnszone_lib = "dnszone debug";
 #else
-const char *dnszone_lib = "dnszone " __DATE__ " " __TIME__ " release";
+const char *dnszone_lib = "dnszone release";
 #endif
 
 dnscore_fingerprint
--- a/sbin/yadifad/confs.c
+++ b/sbin/yadifad/confs.c
@@ -238,10 +238,10 @@
 	    osformatln(termout, "%s %s (%s)\n", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE);
 	    break;
 	case 2:
-	    osformatln(termout, "%s %s (released %s, compiled %s)\n\nbuild settings: %s\n", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE, COMPILEDATE, BUILD_OPTIONS);
+	    osformatln(termout, "%s %s (released %s)\n\nbuild settings: %s\n", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE, BUILD_OPTIONS);
 	    break;
         case 3:
-	    osformatln(termout, "%s %s (released %s, compiled %s)\n", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE, COMPILEDATE);
+	    osformatln(termout, "%s %s (released %s)\n", PROGRAM_NAME, PROGRAM_VERSION, RELEASEDATE);
             yadifad_print_authors();
             break;
 	default:
--- a/sbin/yadifad/confs.h
+++ b/sbin/yadifad/confs.h
@@ -84,7 +84,6 @@
 #define     PROGRAM_NAME                PACKAGE
 #define     PROGRAM_VERSION             PACKAGE_VERSION
 #define     RELEASEDATE                 "2016-11-30"
-#define     COMPILEDATE                 __DATE__
 
     /* List of default values for the different configuration parameters */
 #define     S_CONFIGDIR                 SYSCONFDIR "/"
--- a/bin/yadifa/common-config.h
+++ b/bin/yadifa/common-config.h
@@ -38,7 +38,6 @@
 #define     PROGRAM_NAME                            PACKAGE
 #define     PROGRAM_VERSION                         PACKAGE_VERSION
 #define     RELEASEDATE                             "2016-11-30"
-#define     COMPILEDATE                             __DATE__
 
 #define     ROUND_ROBIN                             0x00
 
