From: Markus Schade <markus.schade@gmail.com>
Date: Wed, 16 Feb 2016 08:30:00 +0100
Subject: fix lintian spelling warning in yadifad binary


--- a/lib/dnsdb/src/journal-cjf-page-output-stream.c
+++ b/lib/dnsdb/src/journal-cjf-page-output-stream.c
@@ -173,7 +173,7 @@
     }
     else
     {
-        log_debug2("cjf: PAGE stream is emtpy");
+        log_debug2("cjf: PAGE stream is empty");
     }
 }
 
--- a/lib/dnsdb/src/journal.c
+++ b/lib/dnsdb/src/journal.c
@@ -275,7 +275,7 @@
 
         journal_mru_size = MAX(MIN(mru_size, ZDB_JOURNAL_FD_MAX), ZDB_JOURNAL_FD_MIN);
         
-        log_debug("journal: intialised with an MRU of %i slots", journal_mru_size);
+        log_debug("journal: initialised with an MRU of %i slots", journal_mru_size);
 
         journal_initialised = TRUE;
     }
--- a/lib/dnsdb/src/zdb_icmtl.c
+++ b/lib/dnsdb/src/zdb_icmtl.c
@@ -953,7 +953,7 @@
         
         if((replay_state & ZDB_ICMTL_REPLAY_COMMIT) != 0)
         {
-            log_info("journal: %{dnsname}: commiting changes", zone->origin);
+            log_info("journal: %{dnsname}: committing changes", zone->origin);
             u64 ts_start = timeus();
             zdb_zone_exchange_locks(zone, ZDB_ZONE_MUTEX_SIMPLEREADER, ZDB_ZONE_MUTEX_LOAD);
             bytearray_input_stream_init_const(&bais, bytearray_output_stream_buffer(&baos), bytearray_output_stream_size(&baos));
@@ -970,19 +970,19 @@
             
             if(ts_delta < 1000)
             {            
-                log_info("journal: %{dnsname}: commited changes (%lluus)", zone->origin, ts_delta);
+                log_info("journal: %{dnsname}: committed changes (%lluus)", zone->origin, ts_delta);
             }
             else if(ts_delta < 1000000)
             {
                 double ts_delta_s = ts_delta;
                 ts_delta_s /= 1000.0;
-                log_info("journal: %{dnsname}: commited changes (%5.2fms)", zone->origin, ts_delta_s);
+                log_info("journal: %{dnsname}: committed changes (%5.2fms)", zone->origin, ts_delta_s);
             }
             else
             {
                 double ts_delta_s = ts_delta;
                 ts_delta_s /= 1000000.0;
-                log_info("journal: %{dnsname}: commited changes (%5.2fs)", zone->origin, ts_delta_s);
+                log_info("journal: %{dnsname}: committed changes (%5.2fs)", zone->origin, ts_delta_s);
             }
                     
             // the current page has been processed
