From: Markus Schade <markus.schade@gmail.com>
Date: Sat, 22 Mar 2014 21:00:00 +0100
Subject: fix lintian whatis warning and FHS in yadifad.conf manpage

--- a/doc/yadifad.conf.man	2013-06-24 14:33:46.000000000 +0200
+++ b/doc/yadifad.conf.man	2014-03-22 20:56:34.741955530 +0100
@@ -12,15 +12,12 @@
 ..
 .TH "YADIFAD\-CONF" "5" "2013\-06\-24" "YADIFA " "YADIFA "
 .SH NAME
-
-.PP
-yadifad.conf
-is the configuration file for \fIyadifad\fP(1)\&.
+yadifad.conf \- configuration file for \fIyadifad\fP(8)\&.
 .PP
 .SH SYNOPSIS
 
 .PP
-${PREFIX}/etc/yadifad.conf 
+/etc/yadifad.conf 
 .PP
 .SH DESCRIPTION
 
@@ -48,7 +45,7 @@
 .PP
 The configuration supports included files.
 .br
-example: include /usr/local/var/zones/master/yadifad\-2.conf 
+example: include /var/cache/yadifa/master/yadifad\-2.conf 
 .PP
 The configuration files can be nested. 
 .PP
@@ -95,7 +92,7 @@
 .PP
 .Vb
 # start yadifad.conf <main> container
-        include /usr/local/etc/yadifad\-main.conf
+        include /etc/yadifa/yadifad\-main.conf
 # end yadifad.conf <main> container
 .Ve
 .PP
@@ -112,24 +109,24 @@
         chroot                  off
 
         # The path of the log files
-        logpath                 "/usr/local/var/log"
+        logpath                 "/var/log/yadifa"
 
         # The path of the pid file
-        pidpath                 "/usr/local/var/run"    
+        pidpath                 "/var/run"    
 
         # The path of the zone files
-        datapath                "/usr/local/var/zones"
+        datapath                "/var/cache/yadifa"
 
         # The path of the DNSSEC keys
-        keyspath                "/usr/local/var/zones/keys"
+        keyspath                "/var/cache/yadifa/keys"
 
         # The path of the transfer and journaling files (AXFR & IXFR)
-        xfrpath                 "/usr/local/var/zones/xfr"
+        xfrpath                 "/var/cache/yadifa/xfr"
 
         # The version returned by a query to version.yadifa. CH TXT
-        version                 "1.0.0rc2"
+        version                 "1.0.3"
 
-        # Enable EDNS0 support (?)
+        # Enable EDNS0 support (option not supported)
         edns0                   on
 
         # Set the maximum UDP packet size.  Packetsize annot be less than 512.  or more than 65535.  A typical choice is 4096.
