From: Markus Schade <markus.schade@gmail.com>
Date: Wed, 16 Feb 2016 08:30:00 +0100
Subject: fix lintian spelling warning in yadifad binary


diff --git a/lib/dnsdb/src/journal-cjf-page-output-stream.c b/lib/dnsdb/src/journal-cjf-page-output-stream.c
index 2a4e28e..34dc959 100644
--- a/lib/dnsdb/src/journal-cjf-page-output-stream.c
+++ b/lib/dnsdb/src/journal-cjf-page-output-stream.c
@@ -172,7 +172,7 @@ journal_cjf_page_output_stream_next(output_stream *stream)
     }
     else
     {
-        log_debug2("cjf: PAGE stream is emtpy");
+        log_debug2("cjf: PAGE stream is empty");
     }
 }
 
diff --git a/lib/dnsdb/src/journal.c b/lib/dnsdb/src/journal.c
index fbc5dc2..3b7db4a 100644
--- a/lib/dnsdb/src/journal.c
+++ b/lib/dnsdb/src/journal.c
@@ -275,7 +275,7 @@ journal_init(u32 mru_size)
 
         journal_mru_size = MAX(MIN(mru_size, ZDB_JOURNAL_FD_MAX), ZDB_JOURNAL_FD_MIN);
         
-        log_debug("journal: intialised with an MRU of %i slots", journal_mru_size);
+        log_debug("journal: initialised with an MRU of %i slots", journal_mru_size);
 
         journal_initialised = TRUE;
     }
diff --git a/lib/dnsdb/src/zdb_icmtl.c b/lib/dnsdb/src/zdb_icmtl.c
index 0172229..4091bb7 100644
--- a/lib/dnsdb/src/zdb_icmtl.c
+++ b/lib/dnsdb/src/zdb_icmtl.c
@@ -808,7 +808,7 @@ zdb_icmtl_replay(zdb_zone *zone)
         
         if((replay_state & ZDB_ICMTL_REPLAY_COMMIT) != 0)
         {
-            log_info("journal: %{dnsname}: commiting changes", zone->origin);
+            log_info("journal: %{dnsname}: committing changes", zone->origin);
             u64 ts_start = timeus();
             zdb_zone_exchange_locks(zone, ZDB_ZONE_MUTEX_SIMPLEREADER, ZDB_ZONE_MUTEX_LOAD);
             bytearray_input_stream_init_const(&bais, bytearray_output_stream_buffer(&baos), bytearray_output_stream_size(&baos));
@@ -825,19 +825,19 @@ zdb_icmtl_replay(zdb_zone *zone)
             
             if(ts_delta < 1000)
             {            
-                log_info("journal: %{dnsname}: commited changes (%lluus)", zone->origin, ts_delta);
+                log_info("journal: %{dnsname}: committed changes (%lluus)", zone->origin, ts_delta);
             }
             else if(ts_delta < 1000000)
             {
                 double ts_delta_s = ts_delta;
                 ts_delta_s /= 1000.0;
-                log_info("journal: %{dnsname}: commited changes (%5.2fms)", zone->origin, ts_delta_s);
+                log_info("journal: %{dnsname}: committed changes (%5.2fms)", zone->origin, ts_delta_s);
             }
             else
             {
                 double ts_delta_s = ts_delta;
                 ts_delta_s /= 1000000.0;
-                log_info("journal: %{dnsname}: commited changes (%5.2fs)", zone->origin, ts_delta_s);
+                log_info("journal: %{dnsname}: committed changes (%5.2fs)", zone->origin, ts_delta_s);
             }
                     
             // the current page has been processed
