From: Markus Schade <markus.schade@gmail.com>
Date: Sat, 24 Nov 2014 21:00:00 +0100
Subject: fix lintian whatis warning in yadifa manpage

--- a/doc/yadifa.man	2014-11-24 20:50:37.378825272 +0100
+++ b/doc/yadifa.man	2014-11-24 20:51:50.202435459 +0100
@@ -12,10 +12,7 @@
 ..
 .TH "YADIFA" "8" "2014\-07\-01" "YADIFA " "YADIFA "
 .SH NAME
-
-.PP
-YADIFA
-is an utility for controlling YADIFAD 
+yadifa \- utility for controlling \fIyadifad\fP(8)\&
 .PP
 .SH SYNOPSIS
 
